config/jenkins_infra.ci.jenkins.io.yaml

```
@@ -31,8 +31,21 @@ controller:
     tag: 1.37.0-2.453
     pullPolicy: IfNotPresent
   nodeSelector:
-    kubernetes.io/os: linux
-    kubernetes.azure.com/agentpool: linuxpool
+    kubernetes.azure.com/agentpool: infracictrl
```

```suggestion
    kubernetes.io/os: linux
    kubernetes.azure.com/agentpool: infracictrl
```

I would keep the OS nodeselector. Not required but makes things explicit